background.js

```diff
@@ -18,4 +18,12 @@ browser.runtime.onMessage.addListener((message, sender, sendResponse) => {
 
     return true;
   }
+  else if (message.cmd === "protectionError") {
```

Comment:
Personal preference, I prefer command to start with a verb e.g. `showDrmError` or `showProtectionError`.